src/Introspection/__Type.hack

```diff
@@ -20,7 +20,7 @@ interface __Type {
     public function getIntrospectionDescription(): ?string;
 
     <<GraphQL\Field('fields', 'Fields of the type, only applies to OBJECT and INTERFACE')>>
-    public function getIntrospectionFields(bool $include_deprecated = false): ?vec<__Field>;
+    public function getIntrospectionFields(bool $includeDeprecated = false): ?vec<__Field>;
```

Comment:
`schema.json` won't generate without this change. (Or we'd need to pass in `include_deprecated` from the client, which seems wrong).

In general it might be nice to support auto-converting from snake case to camel case when converting to and from GraphQL arguments and Hack arguments?

Reply:
It's probably easier/less confusing to adopt the same conventions for GraphQL as what you're using for Hack. There are only a very small number of these names that we can't change because they're required by the GraphQL spec.

Reply:
Makes sense!